### ISSUE TITLE
Selected technique enum doesn't update when changing technique name

### ISSUE THREAD

**Issue by jhnnslmk** [mrvux/dx11-vvvv#302]
Hey master vux,
minor one:
When changing the currently compiled technique's name it doesn't update back in vvvv.
Shader stops drawing (which is fine), pin still shows old name when hovering.
Turning red to prompt the user to re-select another technique would be the most intuitive solution I guess. When exposing the enum to IOBox it turns red as expected.
Doesn't happen for dx9, so figured I report it here.

Thank you!

**Comment by mrvux** [comment]
Indeed, I actually noticed that, seems enum indexing in 4v is kinda broken tho.

Here is a small repro (standalone) that needs to reach forum by the look of it.

[enum_index.zip](https://github.com/mrvux/dx11-vvvv/files/973038/enum_index.zip)

**Comment by mrvux** [comment]
On more elaboration, seems as soon as enum changes and it can't find it anymore, pin changes to index 2 (out of range). but when we change back to an existing value, pin index does not "recover"

**Comment by gregsn** [comment]
please retry with latest alpha. thanks!

**Comment by gregsn** [comment]
please check with latest alpha. index -1 should get returned. close?

**Comment by gregsn** [comment]
well. actually. now it's up. sorry
